src/auth-service/config/constants.js

```diff
@@ -296,6 +296,77 @@ const defaultConfig = {
 
     return projection;
   },
+
+  USERS_INCLUSION_PROJECTION: {
+    _id: 1,
+    firstName: 1,
+    lastName: 1,
```

Comment:
Taking the inclusion and exclusion statements outside the User entity to the constant's file